legacy/util.c

```
@@ -285,3 +285,23 @@ int write_bytes(const uint8_t *src, size_t count, BufferWriter *writer) {
   writer->position += count;
   return 1;
 }
+
+uint64_t deser_compact_size(BufferReader *s) {
```

Cppcheck (2.10-2)

[style] The function 'deser_compact_size' is never used.

(unusedFunction)